plugins/win-capture/graphics-hook/obs-vulkan64.json

```diff
@@ -0,0 +1,49 @@
+{
+    "file_format_version": "1.1.2",
+    "layer": {
+        "name": "VK_LAYER_OBS_HOOK",
+        "type": "GLOBAL",
+        "library_path": ".\\graphics-hook64.dll",
+        "api_version": "1.1.114",
+        "implementation_version": "1",
+        "description": "Open Broadcaster Software hook",
+        "functions": {
+            "vkNegotiateLoaderLayerInterfaceVersion": "OBS_Negotiate"
+        },
+        "instance_extensions": [
+            {
+                "name": "VK_KHR_external_memory_capabilities",
+                "spec_version": "1.1"
+            },
+            {
+                "name": "VK_KHR_get_physical_device_properties2",
+                "spec_version": "1"
+            }
+        ],
+        "device_extensions": [
+            {
+                "name": "VK_KHR_external_memory_win32",
+                "spec_version": "1"
+            },
+            {
+                "name": "VK_KHR_external_memory",
+                "spec_version": "1"
+            },
+            {
+                "name": "VK_KHR_dedicated_allocation",
+                "spec_version": "1"
+            },
+            {
+                "name": "VK_KHR_get_memory_requirements2",
+                "spec_version": "1"
+            },
+            {
+                "name": "VK_KHR_bind_memory2",
+                "spec_version": "1"
+            }
+        ],
```

Comment:
If I understand correctly, these lists are for extensions implemented, not used. I would try removing all of this.